exercises/practice/palindrome-products/.articles/is-palindrome-check/snippet.md

```diff
@@ -0,0 +1 @@
+*I don't really know what this file is for, but the github action is failing without it*
```

Comment:
It's what's displayed on the right hand side of the dig deeper page as a small summary of the article.
See https://exercism.org/docs/building/tracks/practice-exercises#h-file-articles-article-slug-snippet-txt for more information.

Example:

![image](https://user-images.githubusercontent.com/135246/230289929-6de37829-db77-4d9f-bc8b-a6231ec10c41.png)